### DISCUSSION TITLE
In Next 13, how to use next/link with Internationalization?

### DISCUSSION THREAD

**Discussion by robertwbradford** [vercel/next.js#47363] [Help]
### Summary

I see in https://beta.nextjs.org/docs/guides/internationalization that it is easy enough to include an `/app/[lang]` route segment for making the `lang` parameter available to pages, layouts, etc. 

I am not seeing how `next/link` would be aware of Internationalization though. Hopefully we don't have to manually construct URLs to include the language. I don't see any mention in https://beta.nextjs.org/docs/api-reference/components/link. 

I assume this is in the works, but didn't see a place where this was being discussed. Does anyone have insight on this? Thanks!

**Comment by thedv91** [comment]
I am also facing the same problem. Anyone can help me?
